object.c

```diff
@@ -98,6 +98,15 @@ rb_obj_embedded_size(uint32_t numiv)
     return offsetof(struct RObject, as.ary) + (sizeof(VALUE) * numiv);
 }
 
+/*!
+ * :nodoc:
```

Comment:
Since `rb_obj_hide` is not a method implementation, RDoc does not read this comment and this directive does nothing.